packages/jbrowse-plugin-apollo/src/TabularEditor/HybridGrid/HybridGrid.tsx

```diff
@@ -38,14 +59,24 @@ const HybridGrid = observer(function HybridGrid({
 }: {
   model: DisplayStateModel
 }) {
-  const { apolloHover, seenFeatures, selectedFeature, tabularEditor } = model
+  const { apolloHover, lgv, seenFeatures, selectedFeature, tabularEditor } =
+    model
   const theme = useTheme()
   const { classes } = useStyles()
   const scrollContainerRef = useRef<HTMLDivElement>(null)
   const [contextMenu, setContextMenu] = useState<ContextMenuState>(null)
-
+  const { loadingRegions } = getSession(model).apolloDataStore
   const { filterText } = tabularEditor
 
+  // filters seenFeatures by features only directly observed by the user
+  const [visibleRegion] = lgv.getSelectedRegions()
+  const visibleFeatures = [...seenFeatures.entries()].filter((entry) => {
+    const [, feature] = entry
+    return (
+      feature.start > visibleRegion.start && feature.end < visibleRegion.end
```

Comment:
@garrettjstevens 

Adds filtering to the seenFeatures property such that what's displayed on the table is only what is being observed in the graph (previously, it showed everything that had been loaded). "No data" message will display when this value is empty.

Not sure if this is correcting broken behaviour or if it should be kept as showing everything that's currently been loaded. Might be a problem with `model.displayedRegions` ?

Reply:
Re: styling, for the table it might be better to have it centred in the table like a data grid MUI component, subjective though